src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTape.java

```diff
@@ -0,0 +1,273 @@
+     */
+    private static final double WE_END_COLLISION_X = 0.5625;
+
+    /**
+     * Start of the collision box at z facing South/North.
+     */
+    private static final double WE_START_COLLISION_Z = 0.0;
+
+    /**
+     * End of the collision box facing South/North.
+     */
+    private static final double WE_END_COLLISION_Z = 1.0;
+
+    /**
+     * Height of the collision box.
+     */
+    private static final double HEIGHT_COLLISION = 1.0;
+    /**
+     * How much light goes through the block.
+     */
+    private static final int LIGHT_OPACITY = 0;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public BlockConstructionTape()
+    {
+        super(Material.WOOD);
+        initBlock();
+    }
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(), BLOCK_NAME));
+        setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        GameRegistry.register(this);
+        GameRegistry.register((new ItemBlock(this)).setRegistryName(this.getRegistryName()));
+        this.setDefaultState(this.blockState.getBaseState().withProperty(FACING, EnumFacing.NORTH));
+        setHardness(BLOCK_HARDNESS);
+        setResistance(RESISTANCE);
+        setLightOpacity(LIGHT_OPACITY);
+    }
+
+    @Override
+    public boolean isReplaceable(IBlockAccess worldIn, BlockPos pos)
+    {
+        return true;
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @javax.annotation.Nullable
+    @Deprecated
+    @Override
+    public AxisAlignedBB getCollisionBoundingBox(final IBlockState blockState, final IBlockAccess worldIn, final BlockPos pos)
+    {
+        return null;
+    }
+
+    @Override
+    public int getMetaFromState(@NotNull final IBlockState state)
+    {
+        return state.getValue(FACING).getIndex();
+    }
+
+    /**
+     * Convert the given metadata into a BlockState for this Block.
+     *
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @NotNull
+    @Override
+    @Deprecated
+    public IBlockState getStateFromMeta(final int meta)
+    {
+        EnumFacing enumfacing = EnumFacing.getFront(meta);
+
+        if (enumfacing.getAxis() == EnumFacing.Axis.Y)
+        {
+            enumfacing = EnumFacing.NORTH;
+        }
+
+        return this.getDefaultState().withProperty(FACING, enumfacing);
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @SideOnly(Side.CLIENT)
+    @Override
+    @Deprecated
+    public boolean shouldSideBeRendered(IBlockState blockState, IBlockAccess blockAccess, BlockPos pos, EnumFacing side)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1133)